[dynamo] lintrunner for gb_registry adds/updates

Sidharth123-cpu opened the pull request:
This PR adds automation to adding/updating the JSON registry through the lintrunner.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #158460



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @Lucaskabela

Commit:
[dynamo] lintrunner for gb_registry adds/updates

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"




[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Sidharth123-cpu commented:
#suppress-bc-linter

Sidharth123-cpu commented:
suppress-bc-linter

Sidharth123-cpu commented:
@pytorchbot label "suppress-bc-linter"

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 reviewed (commented):
Linter should output that manual intervention may be necessary in ambiguous cases.

We should also have an easily-accessible public facing doc describing the json format so that devs will know what manual format we expect.

Reply: Already discussed and demo'd on Friday's presentation

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Sidharth123-cpu commented:
Test file has been added for lintrunner

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

williamwen42 reviewed (commented):
Thanks for adding thorough tests!

Verified that the gb id linter is being tested in CI: https://github.com/pytorch/pytorch/actions/runs/16461701789/job/46530263687?pr=158460

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames

[ghstack-poisoned]

Sidharth123-cpu commented:
@pytorchbot merge

Commit:
Update on "[dynamo] lintrunner for gb_registry adds/updates"

This PR adds automation to adding/updating the JSON registry through the lintrunner.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang amjames Lucaskabela

[ghstack-poisoned]

Sidharth123-cpu commented:
@pytorchbot merge

Sidharth123-cpu commented:
@pytorchmergebot -f "Unrelated test failure with Inductor; Also got @clee2000 approval as well to force merge"

Sidharth123-cpu commented:
@pytorchbot -f "Unrelated test failure with Inductor; Also got @clee2000 approval as well to force merge"

Sidharth123-cpu commented:
@pytorchbot merge -f "Unrelated test failure with Inductor; Also got @clee2000 approval as well to force merge"
